src/worldcereal/openeo/feature_extractor.py

```diff
@@ -0,0 +1,115 @@
+"""Feature computer GFMAP compatible to compute Presto embeddings."""
+
+import xarray as xr
+from openeo_gfmap.features.feature_extractor import PatchFeatureExtractor
+
+class PrestoFeatureExtractor(PatchFeatureExtractor):
+    """Feature extractor to use Presto model to compute embeddings.
+    This will generate a datacube with 128 bands, each band representing a
+    feature from the Presto model.
+    """
+
+    import functools
+    from pathlib import Path
+    from typing import Tuple
+
+    PRESTO_PATH = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/presto.pt"  # NOQA
```

Comment:
I would put this as optional or even required argument when creating an instance of this class, as there will be multiple (finetuned) Presto model weights. The current one is finetuned for crop/no-crop, and can (for now) be the default, but still we need the option to provide other weights.

Reply:
I added a "presto_url" parameter